routes/customers.js

```diff
@@ -48,16 +50,15 @@ apiRouter.post("/login", async (req, res, next) => {
 
       res.send({
         firstName: user.first_name,
+        lastName: user.last_name,
+        custID: user.cust_id,
         siteAdmin: user.isadmin,
+        custEmail: user.cust_email,
         token,
         cartID: cart.cart_id,
         cart: cartArray,
       });
-    } else {
-      next({
-        name: "IncorrectCredentialsError",
-        message: "Please verify your email and password",
-      });
+
```

Comment:
} else {
      next({
        name: "IncorrectCredentialsError",
        message: "Please verify your email and password",
      });
    }